src/main/java/org/takes/servlet/RqFrom.java

```diff
@@ -57,19 +54,22 @@
     @Override
     public Iterable<String> head() {
         final Collection<String> head = new LinkedList<>();
-        head.add(new RqFrom.HttpHead(this.sreq).toString());
-        head.add(new RqFrom.HttpHost(this.sreq).toString());
-        final Enumeration<String> names = this.sreq.getHeaderNames();
-        while (names.hasMoreElements()) {
-            final String header = names.nextElement();
-            head.add(
+        head.add(new HttpHead(this.sreq).toString());
+        final Collection<String> names = Collections.list(
```

Comment:
@fabriciofx Let's use something from `cactoos` here, I think that `ListOf` should do the trick.

Reply:
@paulodamaso Unfortunately, `ListOf` doesn't work with `Enumeration` therefore I must use `Collections.list`. Ah, I've opened a issue into `cactoos` project to create a new constructor to `ListOf` to support `Enumeration` but it was denied (because is too old).

Reply:
@fabriciofx I've seen the issue in `cactoos`... Well, as they decided to not support `Enumeration` we should let it as is then, thanks.